runtime/doc/treesitter.txt

```diff
@@ -979,6 +982,11 @@ get_filetypes({lang})                *vim.treesitter.language.get_filetypes()*
         (`string[]`) filetypes
 
 get_lang({filetype})                      *vim.treesitter.language.get_lang()*
+    Returns the language name to be used when loading a parser for {filetype}.
+
+    If no language has been explicitly registered via
+    |vim.treesitter.language.register()|, default to {filetype}. For composite
+    filetypes like `html.glimmer`, only the main filetype is returned.
```

Comment:
@zeertzjq I can't find where the help files talk about this feature. Is there some standard terminology I can use instead?

Reply:
It's in Vim's `:h improvements-7` :)

Reply:
Worst documented feature ever ;) I think that should be somewhere in `filetype` -- I've seen repeated misunderstandings of how this works...